ubuntu/standard/Dockerfile.template

```diff
@@ -115,6 +115,11 @@ RUN curl https://packages.microsoft.com/keys/microsoft.asc | gpg --dearmor > /et
 RUN dotnet help
 ENV dotnet=/usr/bin/dotnet
 
+# Install AzCopy (depends on .NET Core)
```

Comment:
Unless we think this tool will be used for other purposes, you might consider to install AzCopy, use it, then uninstall it all within the same RUN statement so that it has no impact on the image.

Reply:
Sure